src/cli/runCli.ts

```diff
@@ -27,7 +27,7 @@ export const runCli = async (
         .option("-V --version", "output the package version");
 
     const parsedArgv = {
-        config: "./eslintrc.js",
+        config: "./.eslintrc.js",
```

Comment:
You'll also want to change the `createTests.ts` default path, and rename the files in `test/tests/*`.

Reply:
I'll do this in #217.